sdk/core/Azure.Core.Experimental/src/DynamicJson.DynamicMetaObject.cs

```diff
@@ -48,11 +51,21 @@ internal object GetProperty(string name)
             return null;
         }
 
+        internal T ConvertTo<T>()
+        {
+            // TODO: Respect user-provided serialization options
+            // TODO: Could we optimize this by serializing from the byte array instead?  We don't currently slice into this in WriteTo(), but could look at storing that.
+            return JsonSerializer.Deserialize<T>(_element.ToString() , MutableJsonDocument.DefaultJsonSerializerOptions);
```

Comment:
Can you use this overload? https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserialize?view=net-7.0#system-text-json-jsonserializer-deserialize(system-text-json-jsonelement-system-type-system-text-json-serialization-jsonserializercontext). This will avoid creating a string.

Reply:
Much better!  Yes, thank you, I absolutely will.

Reply:
Oh, stink, no, it looks like this is only available in .NET 6 and 7.

Reply:
Added multitargeting to address.